querybook/server/lib/ai_assistant/prompts/data_doc_title_prompt.py

```diff
@@ -0,0 +1,19 @@
+from langchain.prompts import PromptTemplate
+
+
+prompt_template = """
+You are a helpful data scientist that can summarize the data doc.
+
+I have querybook data doc with the following data cells. Generate a brief 10-word-maximum title that best summarize them for my document.
+
+===Data Doc Cell Contents
```

Comment:
these are things the AI doesn't need to know, you can say this is a SQL analysis doc